stdlib/test/builtin/test_string.mojo

```diff
@@ -101,6 +101,20 @@ fn test_equality_operators() raises:
     assert_not_equal(s0, "notabc")
 
 
+fn test_compare_operators() raises:
+    assert_true("feefef".__str__().__gt__("feefe".__str__()))
```

Comment:
You can use str() instead of `__str__()`, same for `>` instead of `__gt__`, etc...

Reply:
😊ok, thanks for your advice